### ISSUE TITLE
Need permission to use activity in school

### ISSUE THREAD

**Issue by goathead1NCV** [pamelafox/parallel-demo#9]
Hello,

My name is Jeffrey Gates, and I teach for North Carolina Virtual Public Schools (https://ncvps.org/). I'm planning to use the parallel demo in a lesson in a Computer Science class. My school in very strict about attribution and rights. Even though the link is available on Khan Academy, I would like to get explicit permission to use your website for educational (public education), non-commercial purposes.  Please let me know if you have any questions. jeffrey.gates@ncpublicschools.gov

**Comment by pamelafox** [comment]
Yes, all KA content defaults to CC-NC-BY-SA. I can also add a LICENSE to this readme if they'd like (probably Apache) but it sounds like you're mostly using the KA content.
